doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -783,6 +783,11 @@ Copy its contents to your AArch64 Linux device.
 
 :pencil: If you want a binary for the runtime environment (jre), you must run `make legacy-jre-image`, which produces a jre build in the **build/linux-aarch64-normal-server-release/images/jre** directory.
 
+:pencil: One of the images created with `make all` is the `debug-image`. This directory contains files that provide debug information for executables and shared libraries when using native debuggers.
+To use it, copy the contents of `debug-image` over the jdk before using the jdk with a native debugger.
+Another image created is the `test` image, which contains executables and native libraries required when running some functional and OpenJdk testing.
+When running tests put the test image contents in the `native-test-libs` directory at the same level as TEST_JDK_HOME. See the [OpenJ9 test user guide](https://github.com/eclipse-openj9/openj9/blob/master/test/docs/OpenJ9TestUserGuide.md).
```

Comment:
Copying the contents of the test image is more work than necessary; setting `NATIVE_TEST_LIBS` as discussed in that document is sufficient (and less work).

Reply:
and is setting the LD_LIBRARY_PATH required, or does the harness set that for you once NATIVE_TEST_LIBS is set?